cmd/buildah/rmi.go

```diff
@@ -222,9 +234,35 @@ func untagImage(imgArg string, image *storage.Image, store storage.Store) (strin
 }
 
 func removeImage(image *storage.Image, store storage.Store) (string, error) {
+	parent, err := getParent(store, image.TopLayer)
+	if err != nil {
+		return "", err
```

Comment:
will err be sufficient to let the user know why the removal didn't happen?  Should we have an errros.Wrapf here?

Reply:
same comment for other returns below.

Reply:
wrapped the errors.